test/functional/lib/ipfs-companion.test.js

```diff
@@ -1,31 +1,36 @@
-const { describe, it, before, beforeEach, after } = require('mocha')
-const { expect } = require('chai')
-const browser = require('sinon-chrome')
-const { URL } = require('url')
-const { optionDefaults } = require('../../../add-on/src/lib/options')
-const { AbortController } = require('abort-controller')
+import { describe, it, before, beforeEach, after } from 'mocha'
+import { expect } from 'chai'
+import browser from 'sinon-chrome'
+import { URL } from 'url'
+import { AbortController } from 'abort-controller'
+import { optionDefaults } from '../../../add-on/src/lib/options.js'
+browser.runtime.id = 'testid'
+global.browser = browser
+global.chrome = browser
+global.navigator = {
+  clipboard: {
+    writeText: () => {}
```

Comment:
this was something extra I had to add to prevent dependency from failing.

Reply:
which dependency? Can we add a comment to the code to call it out? Do we need that dependency?